js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -40,6 +42,8 @@ export default class HowTo extends Component {
 	constructor( props ) {
 		super( props );
 
+		const applyFilters = get( window, "wp.hooks.applyFilters", noop );
```

Comment:
question: I'm not familiar with the way lodash/get works, but is this basically equivalent to getting something from the global `window.wp` without checking if it exists before?

Reply:
What happens is that, if `wp.hooks.aplyFilters` doesn't exist, `noop` is used as a fallback. Right now, if it isn't found, the defaultDurationText becomes empty as a result because the default of applyFilters isn't set as well. so I need to address that.